Revise the expressions that can be pushed down (#7939)
pingcap/docs: functions-and-operators/expressions-pushed-down.md

## Blocklist specific expressions

> Suppose that unexpected behavior occurs in the calculation process when pushing down the [supported expressions](#supported-expressions-for-pushdown) or specific data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)). In this case, you can restore the application quickly by prohibiting the pushdown of the corresponding functions, operators, or data types. Specifically, you can prohibit the functions, operators, or data types from being pushed down by adding them to the blocklist `mysql.expr_pushdown_blacklist`. For details, refer to [Add to the blocklist](#add-to-the-blocklist).

Reviewer: ```suggestion
If unexpected behavior occurs in the calculation process when pushing down the [supported expressions](#supported-expressions-for-pushdown) or specific data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)), you can restore the application quickly by prohibiting the pushdown of the corresponding functions, operators, or data types. Specifically, you can prohibit the functions, operators, or data types from being pushed down by adding them to the blocklist `mysql.expr_pushdown_blacklist`. For details, refer to [Add to the blocklist](#add-to-the-blocklist).
```

Field description:

+ `name`: the name of the function, operator, or data type that is prohibited from being pushed down.
+ `store_type`: specifies to which storage engine the function, operator, or data type is prohibited from being pushed down. Currently, TiDB supports the three storage engines: `tikv`, `tidb`, and `tiflash`. `store_type` is case-insensitive. If a function is prohibited from being pushed down to multiple storage engines, use a comma to separate each engine.
+ `reason` : The reason why the function is blocklisted.

### Add to the blocklist

> To add one or more [functions, operators](#supported-expressions-for-pushdown), or data types (**only** [`ENUM` type](/data-type-string.md#enum-type) and [`BIT` type](/data-type-numeric.md#bit-type)) to the blocklist, perform the following steps:

Reviewer: ```suggestion
To add one or more [functions, operators](#supported-expressions-for-pushdown), or data types (**only** the [`ENUM` type](/data-type-string.md#enum-type) and the [`BIT` type](/data-type-numeric.md#bit-type)) to the blocklist, perform the following steps:
```
